src/useAnimation.ts

```diff
@@ -210,6 +210,15 @@ export function useAnimation(
           previousWeekdaysEl.style.opacity = "0";
         }
 
+        const animatedButtons = previousMonthEl.querySelectorAll(
+          "[data-animated-button]"
```

Comment:
Is it possible to remove this logic due to the latest changes pushed to the branch?

If we still need, `data-animated-button` seems a bit too generic, as there are other buttons rendered by the day picker, what do you think of adding a bit more context like `data-animated-nav-button`?
Also, add the `data` attribute to all types of layouts to keep it consistent.

We can add an assertion to the `useAnimation` test (I can do that if we need to keep this logic).